### ISSUE TITLE
Missing dependencies in NPM registry.

### ISSUE THREAD

**Issue by slashbrackets** [angular/angular-cli#2184]
> Please provide us with the following information:
> ---------------------------------------------------------------

1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
Windows 10

2. Versions. Please run `ng --version`. If there's nothing outputted, please run
in a Terminal: `node --version` and paste the result here:
Uninstalled angular-cli and tried to install latest.

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
No code yet. Just angular-cli install issues.

4. The log given by the failure. Normally this include a stack trace and some
   more information.
3197 silly gunzTarPerm extractEntry bin/common/bootstrap.js
3198 silly gunzTarPerm modified mode [ 'bin/common/bootstrap.js', 438, 420 ]
3199 silly gunzTarPerm extractEntry bin/common/parse-cli-args.js
3200 silly gunzTarPerm modified mode [ 'bin/common/parse-cli-args.js', 438, 420 ]
3201 error 404 Not Found
3201 error 404
**3201 error 404 'angular/compiler' is not in the npm registry.**
3201 error 404 You should bug the author to publish it
3201 error 404 It was specified as a dependency of 'angular-cli'
3201 error 404
3201 error 404 Note that you can also install from a
3201 error 404 tarball, folder, or http url, or git url.
3202 error System Windows_NT 10.0.10586

5. Mention any other details that might be useful.

I've tried:
`npm uninstall -g angular-cli`

`npm cache clean`

`npm install -g angular-cli@latest` and `npm install -g angular-cli@1.0.0-beta.14` and `npm install -g angular-cli@webpack`

Always error with some dependency not being in the npm registry. Sometimes it's `angular/common` or `angular-cli/ast-tools` or `angular\compiler` or it seems some other random dependency.

**Comment by slashbrackets** [comment]
**Update:**
I've tried:
`npm uninstall -g angular-cli`

`npm cache clean`

`npm install -g angular-cli@latest` and `npm install -g angular-cli@1.0.0-beta.14` and `npm install -g angular-cli@webpack`

Always error with some dependency not being in the npm registry. Sometimes it's `angular/common` or `angular-cli/ast-tools` or it seems some other random dependency.

**Comment by filipesilva** [comment]
What are your versions of node and npm? You can do `node -v && npm -v` to get them both.

**Comment by slashbrackets** [comment]
I had a pretty hard time upgrading npm. I was on version 1.4.9 and it was persistent no matter what I did. Uninstalled nodejs, and re-installed, etc. And deleted folders, etc. Nothing worked. Eventually using where.exe npm I found that there was an instance of npm under chocolatey. So I uninstalled chocolatey and reinstalled nodejs and that solved the issue.

Now that I have the latest node and npm and was able to install angular-cli fine.
